bitnami/pytorch/README.md

```diff
@@ -0,0 +1,161 @@
+| `global.imagePullSecrets`              | Global Docker registry secret names as an array                                              | `[]` (does not add image pull secrets to deployed pods) |
+| `image.registry`                       | PyTorch image registry                                                                       | `docker.io`                                             |
+| `image.repository`                     | PyTorch image name                                                                           | `bitnami/pytorch`                                       |
+| `image.tag`                            | PyTorch image tag                                                                            | `{VERSION}`                                             |
+| `image.pullPolicy`                     | Image pull policy                                                                            | `IfNotPresent`                                          |
+| `image.pullSecrets`                    | Specify docker-registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `image.debug`                          | Specify if debug logs should be enabled                                                      | `false`                                                 |
+| `git.registry`                         | Git image registry                                                                           | `docker.io`                                             |
+| `git.repository`                       | Git image name                                                                               | `bitnami/git`                                           |
+| `git.tag`                              | Git image tag                                                                                | `latest`                                                |
+| `git.pullPolicy`                       | Git image pull policy                                                                        | `Always`                                                |
+| `git.pullSecrets`                      | Specify docker-registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `pytorch.entrypoint.file`              | Main entrypoint to your application                                                          | `main.py`                                               |
+| `pytorch.entrypoint.args`              | Args required by your entrypoint                                                             | `nil`                                                   |
+| `pytorch.distributed.enabled`          | Enable distributed mode for PyTorch                                                          | `true`                                                  |
```

Comment:
What about using a parameter such as: **"mode"** that accepts two possible values: **"standalone"** or **"distributed"**?

By the way, is "distributed" a common name on PyTorch terminology or do they user other names such as "cluster" or "replicaset"?

Reply:
Yes, it is: https://pytorch.org/docs/stable/distributed.html